projects/azure/azure-cicd/secret-refresh-job.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1beta1
+kind: CronJob
```

Comment:
Why not a Prow periodic? With pod utils you get logs too

Reply:
You need access to the ci namespace to manage the token in case it needs to be renewed manually. We are using a separate namespace so that everyone in our team can manage all resources.

Reply:
Dunno :) Now all code in one place, maybe just about that :)

Reply:
OK

Reply:
Can we not use the azure namespace as a build cluster? But in the end whatever works is good -- just figured the logging etc would be nice for yall

Reply:
Yeah, that's possible. We can look into it in a follow-up (need to evaluate our cronjobs on a case by case since some may not need to be exposed).

Reply:
Unfortunately we cannot use the build cluster feature to configure plank with a different namespace - opened https://github.com/kubernetes/test-infra/issues/8862 upstream